src/playlist-maker.js

```diff
@@ -340,6 +340,16 @@ export default function factory(player, initialList, initialIndex = 0) {
         list[playlist.currentIndex_]
       );
 
+      // When playing multiple videos in a playlist the videojs PosterImage
+      // will be hidden using CSS. However, in some browsers the native poster
+      // attribute will briefly appear while the new source loads. Prevent
+      // this by hiding every poster after the first play list item. This
+      // doesn't cover every use case for showing/hiding the poster, but
+      // it will significantly improve the user experience.
+      if (index > 0) {
```

Comment:
I do think we would want to account for `autoadvance > 0` cases, at least. It's likely to come up. This may just mean adding `autoadvance === 0` as part of this condition.

Reply:
Thanks. That makes sense. I wasn't sure how often this configuration parameter was used. I'll add the extra condition.

Reply:
I've tested how this works in practice. If `autoadvance` is set to > 0 the poster isn’t displayed at the end of the video. It waits on the last frame of the video. So if you add  `autoadvance === 0` you still get a poster flash between playlist items.